LICENSE_FILE_HEADER

```diff
@@ -0,0 +1,15 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+//
```

Comment:
Should we add a reference to the actual license file here?
If not a URL then state that the file is to be found at the root of the Kokkos source dir

Reply:
yeah we could add that, just wasn't sure what a stable URL for that is?

Reply:
I think the best would be to upload to kokkos.org